### ISSUE TITLE
To have power turned on for the light posts around the map(s).

### ISSUE THREAD

**Issue by HamzaXSniper** [UncreatedStaff/UncreatedWarfare#131]
A (perhaps destructible) electricity system to enable light posts/garage fences, I get there is an issue of lag. But if using a grid system to divide up the map into sections and when there is a cap in that section it would be enabled. This would be obviously disabled during insurgency gamemode to not give a general location of the caches(or enabled to do exactly that).

**Comment by Samanthuhh** [comment]
I agree, i actually thought about this before, It would bring realism and light graphics into play making the game look far better
